packages/app-distribution/lib/modular/index.js

```diff
@@ -0,0 +1,44 @@
+import { firebase } from '..';
+
+/**
+ * @param {import('@react-native-firebase/app').ReactNativeFirebase.FirebaseApp | undefined} app
+ * @returns {import('.').AppDistribution}
+ */
+export function getAppDistribution(app) {
```

Comment:
All of the APIs are showing up with codecov annotations as "untested".

- did you notice that?
- do you have any theories on that?

I've rebased the PR against current main and will re-push it which will re-trigger CI, maybe that will clear it. But I'm surprised that the e2e tests *appear* to be exercising it above yet the annotations of being untested are here with no explanation in the PR description 🤔